### PR TITLE
[mqtt-cpp] Fix dependencies list

### PR DESCRIPTION
**Pull request by anion155**
Add proper dependencies to library.

### CODE REVIEW

File: ports/mqtt-cpp/CONTROL [hunk 1]
```diff
@@ -2,4 +2,4 @@ Source: mqtt-cpp
 Version: 7.0.1
```
Comment (c72578) [inline ports/mqtt-cpp/CONTROL]: The version in the CONTROL file needs to be updated to:
`Version: 7.0.1-1`

**Comment by anion155** [comment]
@c72578 done

**Comment by NancyLi1013** [comment]
Hi @anion155 Thanks for this PR.
Could you please open an issue for this PR?

**Comment by anion155** [comment]
#9893

**Comment by NancyLi1013** [comment]
Thanks for your update.
Could you please resolve the conflicts?

**Comment by anion155** [comment]
@NancyLi1013 done

**Comment by NancyLi1013** [comment]
Thanks for your update.
It seems that there is something wrong with Linux platform on CI system.
I will re run it once linux pipeline works normally.

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
@anion155 thanks for your PR.
LGTM now.

**Comment by ras0219-msft** [comment]
LGTM, thanks for fixing this up!

**Pull request merged**